app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def most_recent
+    posts.order(created_at: :desc).limit(5)
```

Comment:
- [ ] I think you are missing something here. If you realize, you are using the same method for this model and for the user. For the post model, you are supposed to show the most recent comments, not the most recent posts. So please work on this issue.